doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1624,9 +1624,11 @@ previous :token:`i_item` have been performed.
 
 The second entry in the :token:`i_view` grammar rule,
 ``/ltac:(`` :token:`tactic` ``)``, executes :token:`tactic`.
-Notations can be used to name tactics,  for example::
+Notations can be used to name tactics,  for example
 
-    Notation myop := (ltac:(some ltac code)) : ssripat_scope.
+.. coqtop:: in fail
+
+   Notation "'myop'" := (ltac:(my ltac code)) : ssripat_scope.
```

Comment:
I don't understand this PR. What is your goal? `in fail` is usually not a good idea because it means that failures will be allowed but not shown. `fail` should only be used when a code is purposely failing to show an error to the user (in which case it should use the `all` option).

Reply:
I put the `fail` because `my ltac code` is not valid code (that was there originally). The only point is to change `myop` to `"'myop'"` because otherwise it does not work with scopes to my knowledge...
Of course if it seems better to change `my ltac code` to something valid, I can do.

Reply:
OK thanks, I had missed the other change!

Reply:
Indeed, it would be better to make the code example pass. If you want, one way to achieve this would be some preliminary `coqtop` block:

```
.. coqtop:: none

   Tactic Notation "my" "ltac" "code" := idtac.
```

This should allow removing the `fail` (actually replacing it with `warn` since apparently this still gives the following warning: "This notation contains Ltac expressions: it will not be used for printing. [non-reversible-notation,parsing]").